looks good, though test fail looks real:
```
======================================================================
ERROR: testBadAggregation (corehq.apps.userreports.tests.test_columns.TestFieldColumn)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/tests/test_columns.py", line 55, in testBadAggregation
    "type": "field",
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/reports/factory.py", line 131, in from_spec
    str(e),
BadSpecError: Problem creating column from spec: {
  "field": "doc_id", 
  "display": "doc_id", 
  "type": "field", 
  "aggregation": "simple_", 
  "column_id": "doc_id"
}, message is: u'simple_' not in choices: ['count', 'count_unique', 'min', 'simple', 'max', 'sum', 'avg', 'year', 'month']
```

--- a/corehq/apps/userreports/tests/test_columns.py
+++ b/corehq/apps/userreports/tests/test_columns.py
@@ -1,6 +1,5 @@
 import uuid
 
-from jsonobject.exceptions import BadValueError
 from sqlagg import SumWhen
 from django.test import SimpleTestCase, TestCase
 
@@ -10,6 +9,7 @@ from corehq.apps.userreports.models import (
     DataSourceConfiguration,
     ReportConfiguration,
 )
+from corehq.apps.userreports.exceptions import BadSpecError
 from corehq.apps.userreports.reports.factory import ReportFactory, ReportColumnFactory
 from corehq.apps.userreports.reports.specs import FieldColumn, PercentageColumn, AggregateDateColumn
 from corehq.apps.userreports.sql import IndicatorSqlAdapter
@@ -48,7 +48,7 @@ class TestFieldColumn(SimpleTestCase):
         self.assertEqual('doc_id', field.column_id)
 
     def testBadAggregation(self):
-        with self.assertRaises(BadValueError):
+        with self.assertRaises(BadSpecError):
             ReportColumnFactory.from_spec({
                 "aggregation": "simple_",
                 "field": "doc_id",
@@ -70,7 +70,7 @@ class TestFieldColumn(SimpleTestCase):
             ))
 
     def testBadFormat(self):
-        with self.assertRaises(BadValueError):
+        with self.assertRaises(BadSpecError):
             ReportColumnFactory.from_spec({
                 "aggregation": "simple",
                 "field": "doc_id",
@@ -316,18 +316,18 @@ class TestPercentageColumn(SimpleTestCase):
             "field": "is_pregnant",
             "type": "field",
         }
-        with self.assertRaises(BadValueError):
+        with self.assertRaises(BadSpecError):
             ReportColumnFactory.from_spec({
                 'type': 'percent',
                 'column_id': 'pct',
             })
-        with self.assertRaises(BadValueError):
+        with self.assertRaises(BadSpecError):
             ReportColumnFactory.from_spec({
                 'type': 'percent',
                 'column_id': 'pct',
                 'numerator': field_spec,
             })
-        with self.assertRaises(BadValueError):
+        with self.assertRaises(BadSpecError):
             ReportColumnFactory.from_spec({
                 'type': 'percent',
                 'column_id': 'pct',
@@ -341,7 +341,7 @@ class TestPercentageColumn(SimpleTestCase):
             "field": "is_pregnant",
             "type": "percent",
         }
-        with self.assertRaises(BadValueError):
+        with self.assertRaises(BadSpecError):
             ReportColumnFactory.from_spec({
                 'type': 'percent',
                 'column_id': 'pct',

